client/src/components/Applications/AppForm/fixtures/AppForm.allocations.fixture.js

```diff
@@ -1,6 +1,7 @@
 const allocationsFixture = {
   hosts: {
     'ls5.tacc.utexas.edu': ['TACC-ACI'],
```

Comment:
Guess we can remove all the old systems like ls5, longhorn, and maverick now

Reply:
I updated this a bit.